org-roam.el

```diff
@@ -726,6 +728,10 @@ tag."
                     (buffer-file-name)))
          nil)))))
 
+(defun org-roam--extract-tags-vanilla (_file)
+  "Extract vanilla `org-mode' tags."
```

Comment:
Can we add some details here about what tags are extracted?

This would be all tags in the file, including filetags, and tags in headlines within the file.